device/accton/x86_64-accton_csp7551-r0/platform.json

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
Since `platform.json` is provided, are you planning to provide `hwsku.json` as well? These two can replace `port_config.ini`.